### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by rustytravis** [jekhokie/raspberry-noaa-v2#549] [Ideas]
An excellent product, thank you. I feel an improvement can be made on the Admin tab, in both Passes/Captures,  by replacing the red "X" deletion icons with check-boxes, allowing the opportunity to check all if desired before deletion.

The current red "X" icons are rather clumsy on a laptop with touchpad.

Also, a config/settings.yml option to disable the solar graphics on the page would be very welcomed.

Again, thank you for a very excellent product.

**Comment by dom-robinson** [comment]
Good feature request. thankyou. Can't promise timescale but ill label it so others can spot it too! Thankyou!
